ui/src/album/AlbumGridView.js

```diff
@@ -85,7 +103,7 @@ const useCoverStyles = makeStyles({
 const getColsForWidth = (width) => {
   if (width === 'xs') return 2
   if (width === 'sm') return 3
-  if (width === 'md') return 4
+  if (width === 'md') return 5
```

Comment:
Any particular reason to change this?

Reply:
There is no particular reason; actually, while debugging, I had made these changes and forgot to change them back; I will restore it to the previous value; sorry for the trouble, and thanks for your review.